### ISSUE TITLE
Review/update Redirect implementation documentation

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#18782]
## Description

_**Although the support teams may begin implementing their own redirects in the future, Mikki has asked the IA team to remove the implementation documentation from their draft and focus only on the IA guidance for when and why to do a redirect.**_

The plan to move the documentation to the Platform site has not changed but the task has shifted a bit. 

We should disregard the Guidance/Standards version, and will  need to update the issue accordingly to ensure existing documentation is up to date, accurate, and explicit about guidance on when and why to do redirects.

This documentation will be moved into the Platform Documentation site once our review and update is complete.

As appropriate, rather than being removed, areas of the documentation (i.e., the [Redirect Implementation Workflow](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/redirect-implementation-strategy.md#implementation-by-type) diagram) can be labeled as "Current process" followed by placeholder areas for "Future process" to retain historical knowledge. 

CAIA will be deprecating existing [Redirect Implementation Strategy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/redirect-implementation-strategy.md#redirect-implementation-strategy) documentation once the DEPO content website contains the information. 

## Acceptance criteria
- [x] Review ~[CAIA's Google doc](https://docs.google.com/document/d/1wRcgJn7p4Cuz4cFr-WF3JlvttK5CAbenZkJM4EqBLIs/edit#heading=h.efzncwe2ayez) for accuracy & edits
   - [x] Update https://github.com/department-of-veterans-affairs/va.gov-team/issues/83498 to make CAIA aware of any edits that have been proposed to the Google doc
  - [x] References to dependencies on Public Websites have been removed and/or replaced with placeholder text if the responsible party is not known
- [x] Review the [Github Redirect strategy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/redirect-implementation-strategy.md) for accuracy & edits
- [x]  [Redirect Intake Request template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+franecross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+sitewide%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request) has been reviewed for accuracy and updated to reflect existing processes and considerations to support teams who are implementing their own redirect for the first time.
- [x] Links to other documentation or references are valid

**Comment by jilladams** [comment]
I think we could consider some of this already done, via my side conversations with CAIA. 

CAIA provided this Google doc for review, and my edits are included: https://docs.google.com/document/d/1wRcgJn7p4Cuz4cFr-WF3JlvttK5CAbenZkJM4EqBLIs/edit#heading=h.efzncwe2ayez

CAIA has been trafficking edits on that doc via https://github.com/department-of-veterans-affairs/va.gov-team/issues/83498, and the edits we made in it have been fed into the content updates with the Platform Content team. 

As far as I'm aware, the intent is to deprecate the Github version of redirect documentation once the updated Platform Content site pages go live. If we do further edits now, we need to figure out cycling those changes through CAIA > Platform Content to make sure they make it into the planned updates to the Content site. I feel pretty confident in the feedback I've given, but if the concern is getting engineering eyes on it, maybe we could get Platform Content to provide a draft we can review so we're revising latest for publish?

**Comment by randimays** [comment]
Requested access to CAIA's Google Doc

**Comment by jilladams** [comment]
Shared it with you. However, noting: the technical guidance did get pulled out of that Google Doc. It's worth looking over and commenting on any flags about incorrect technical things. But: the technical information is now updated here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/redirect-implementation-strategy.md, and that's probably a better place to spend your time on technical review.

**Comment by randimays** [comment]
@jilladams Finished reviewing that doc. Very tiny adjustments made in a commit here: https://github.com/department-of-veterans-affairs/va.gov-team/commit/348fc01dc9531c88771af4f2015396ca279f5e59

Great work and THANK YOU for (as always) maintaining these super helpful docs for us!

One question. The welcome modals example for `explore.va.gov` does not show a modal for me when navigating to https://www.va.gov/?from=explore.va.gov. Is there another example we can use, maybe?

<img width="700" alt="Screenshot 2024-08-01 at 1 24 47 PM" src="https://github.com/user-attachments/assets/323644b9-30f2-4b7b-b117-40d86f069481">

<img width="1341" alt="Screenshot 2024-08-01 at 1 29 54 PM" src="https://github.com/user-attachments/assets/99d193d0-b8d2-4a88-af9f-0c1f07a47be3">

**Comment by jilladams** [comment]
Interesting callout - that note about the modal was from the team prior to our contract, and we haven't done a new full subdomain redirect since my time here. I suspect there is/was an expiration on the modal after the subdomain had been redirected for long enough time to no longer be news. 

@mmiddaugh are you familiar with the concept of this modal, maybe from the VAMC migration days? The documentation is saying that a modal might be shown if the param on the URL is from a redirect, e.g. https://www.va.gov/?from=explore.va.gov, but I've never seen this in use. 

If it's not around anymore / we don't foresee using it again, we can remove that note from the docs.

**Comment by randimays** [comment]
@FranECross Forgot to add here that this was pulled in as stretch for sprint 9.

**Comment by randimays** [comment]
> @mmiddaugh are you familiar with the concept of this modal, maybe from the VAMC migration days? The documentation is saying that a modal might be shown if the param on the URL is from a redirect, e.g. https://www.va.gov/?from=explore.va.gov, but I've never seen this in use.

From scrum: Michelle advocates for removing this guidance since we don't have a live example at the moment and don't want to cause confusion.

**Comment by randimays** [comment]
Updated doc to remove this guidance. [Commit here](https://github.com/department-of-veterans-affairs/va.gov-team/commit/af251c0a90cf992b5fac86b732db858745fa529f). @mmiddaugh to look into next steps on handoff. Will leave this ticket open to tie up any loose ends on PW side.

**Comment by FranECross** [comment]
Just noting that Mikki posted in VFS-all-teams that [Guidance on when to use redirects](https://design.va.gov/components/url-standards/redirects) has been added to the URL documentation in the design system.

**Comment by jilladams** [comment]
Closing per sprint planning signoff from Michelle.
